lms/static/js/dashboard/views/course_list_tabbed_view.js

```diff
@@ -0,0 +1,101 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'underscore', 'gettext',
+        'js/dashboard/views/course_list_view',
+        'js/dashboard/views/course_view',
+        'js/components/tabbed/views/tabbed_view'
+    ], function (Backbone, _, gettext, CourseListView, CourseView, TabbedView) {
+
+        var CourseListTabbedView = Backbone.View.extend({
+
+            el: '.wrapper-header-courses',
+
+            courses: {
+                active: {
+                    title: gettext('Current'),
+                    template: '#active-courses-tpl',
+                    url: '/active-courses',
+                    index: 0
+                },
+                completed: {
+                    title: gettext('Archived'),
+                    template: '#completed-courses-tpl',
+                    url: '/completed-courses',
+                    index: 1
+                }
+            },
+
+            initialize: function () {
+                this.setElement(this.el);
+
+                this.activeCourseListView = new CourseListView({
+                    template: this.courses.active.template,
+                    itemViewClass: CourseView,
+                    parent: this
+                });
+                this.completedCourseListView = new CourseListView({
+                    template: this.courses.completed.template,
+                    itemViewClass: CourseView,
+                    parent: this
+                });
+
+                // This slightly tedious approach is necessary
+                // to use regular expressions within Backbone
+                // routes, allowing us to capture which tab
+                // name is being routed to.
+                var router = new Backbone.Router();
```

Comment:
All vars should be declared at the top of a JavaScript function (because hoisting https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting)